import { catchAsyncErrors } from "./catchAsyncErrors";

export const isAuthenticated = catchAsyncErrors(asy)